src/components/ContentUploader/ItemAction.js

```diff
@@ -24,15 +25,17 @@ type Props = {
 const ItemAction = ({ status, onClick, intl }: Props) => {
     let icon = <IconCross />;
     let title = intl.formatMessage(messages.cancel);
+    let tooltip = intl.formatMessage(messages.uploadsCancelButtonTooltip);
 
     switch (status) {
         case STATUS_COMPLETE:
-            icon = <IconCheck />;
+            icon = <IconCheck color='#26C281' />;
```

Comment:
We can just make #26C281 the default color in IconCheck

Reply:
since now we're using icons from 'box-react-ui', keeping it here for now, until we change the default color inside 'box-react-ui'

Reply:
I see, can you move this to a constant at the top?

Reply:
done